src/App.jsx

```diff
@@ -14,57 +16,63 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const clearSelection = () => {
+    setSelectedGood('');
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {!selectedGood ? 'No goods selected' : `${selectedGood} is selected`}
+        <button
+          onClick={clearSelection}
+          data-cy="ClearButton"
+          type="button"
+          className="delete ml-3"
+        />
+      </h1>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+      <table className="table">
+        <tbody>
+          {goods.map(good => {
+            const isSelected = good === selectedGood;
+            const onSelect = setSelectedGood;
```

Comment:
Why you need this?

Reply:
i need this so i dont render the new button, but change data-cy and content (not just classes)

Reply:
But why you need `const onSelect = setSelectedGood;` ?
Why not to just use `setSelectedGood` ?